handler/src/main/java/io/netty/handler/ssl/OpenSslServerContext.java

```diff
@@ -346,65 +350,27 @@ private OpenSslServerContext(
         // Create a new SSL_CTX and configure it.
         boolean success = false;
         try {
-            checkKeyManagerFactory(keyManagerFactory);
             checkNotNull(keyCertChain, "keyCertChainFile");
             checkNotNull(key, "keyFile");
 
-            if (keyPassword == null) {
-                keyPassword = "";
-            }
-
+            keyPassword = keyPassword == null ? StringUtil.EMPTY_STRING : keyPassword;
```

Comment:
it seems like `buildKeyManagerFactory` already accounts for a `null` key ... i wonder if this is redundant and also if this prevents us from distinguishing between no password, and empty password at the lower levels.